### ISSUE TITLE
The underscore function is automatically global

### ISSUE THREAD

**Issue by gflohr** [mamod/JavaScript-Duktape#39]
All Perl symbols that consist of punctuation  only are automatically global. That means that you do not have to export `_()`.

I think that this behavior should at least be documented. And maybe you can think of a better function name so that the global namespace does not get polluted.

If you are searching for another function name, please keep in mind that the double underscore `__()` is already exported by https://metacpan.org/pod/Locale::TextDomain and in relatively wide use.

**Comment by mamod** [comment]
You're totally right, this was a bad decision, it does not have much use anyway so I'll try to deprecate it or think of a better way, I'll keep this open until I sort this issue

Thank you for reporting this
